README.md

```diff
@@ -37,6 +37,7 @@ We currently provide the following resources, broadly organized by topic.
 ### Script Optimization Techniques:
 * [Reducing Plutus Script Sizes](scriptsize.md)
 * [Optimizations to reduce CPU and Mem consumption](scriptmem.md)
+* [Shrinker](https://github.com/Geometer1729/Shrinker)
```

Comment:
Should we place this repo under github.com/Plutonomicon ?

cc @Fresheyeball @Benjmhart

Reply:
I think that makes sense. Are you able to add me to the Plutonomicon group so I can move it?

Reply:
@Geometer1729 It doesn't seem like I have permissions for that, but I've created https://github.com/Plutonomicon/Shrinker and invited you with Write role.

Reply:
Thanks